internal/ui_screen.go

```diff
@@ -38,14 +34,15 @@ const (
 
 const scaleEnvKey = "FYNE_SCALE"
 const defaultScreenName = ":0"
+const baselineDPI = 120.0
```

Comment:
Could you move baselineDPI to config.go and add a descriptive comment?

Reply:
I don't know if it is necessary to move it. As i see it, it has no meaning outside of scaling logic.

Reply:
Fair enough, I'll let it slide since it's a const, but we should stride for central management of all "config-like" options in the future.